### ISSUE TITLE
How to use with beets?

### ISSUE THREAD

**Issue by SuperFola** [IrosTheBeggar/mStream#369]
I'm currently in the process of deploying mStream to use with the database beets created for me, but after searching on both repositories and documentation, I can not find where and how to specify to mStream that it should use a pre-generated database.

I have node 18 and npm 8.8.0, have run `npm run-script wizard`, but it didn't add the mstream command.

Any help would be appreciated.

**Comment by StevenPolley** [comment]
I don't think you can use the beets database directly.  At one point in time, it looks like that was the case, however, I came across this as the latest mention of "beet" in the repository's history and it looks like support was removed due to mstream requiring a more sophisticated data model.  

1b924824a30c859ad87fd1ba5f58f9f51176f0a7

From what I could find, there is just a link in the beets documentation referencing this project, as they happen to work nicely together, each fulfilling different roles with no overlap.  At least that's what's inferred from this conversation: https://github.com/beetbox/beets/issues/2700

TLDR, there is no formal integration but they operate side by side nicely.  I do wish mStream would also link to the beets project however.
